config/types/etcd.go

```diff
@@ -32,7 +33,9 @@ var (
 )
 
 // Options can be the options for any Etcd version
```

Comment:
Not part of this PR, but "etcd" is always lowercase.

Reply:
whoops.